FirebaseStorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 10.3.0
+- [fixed] Use dedicated queue for Storage uploads and downloads instead of global queue. Fixes regression
```

Comment:
What do you think about changing this to "Use serial queue for Storage uploads and downloads instead of a (concurrent) global queue" since I think the serial aspect is most important vs. global?

Reply:
Good point. I saw your PR right after merging.  I updated the PR description and will send a quick followup PR. Thanks!